Built outline components for Ratings and Reviews

aboussarath opened the pull request:
- Built out my components and added my section to app.jsx
- Also added the tiniest amount of layout styling to my section for visibility

sharplessHQ reviewed (commented):
Look good to me via static review :-)

kingsleyallen commented:
Nice work! I like it. 

A couple of notes/questions.

**Top-Level Div**
This is the current structure in `ratings.jsx`:

```
<div>
  <p>Ratings and Reviews</p>
  <div className='ratings-reviews'>
    <Breakdown />
    <ReviewList />
  </div>
</div>
```
Is there a reason for the outer `<div>`? If not, you might want to remove it, make the ratings-reviews `<div>` the top level, and move the `<p>` inside it.

I mention this because it seems like we should aim for a final rendered DOM where all the top-level component divs are siblings just one level below app like this: 
```
<div id="app">
  <div>...</div>   <-- overview
  <div>...</div>   <-- related products
  <div>...</div>   <-- questions and answers
  <div>...</div>   <-- ratings and reviews
</div>
```

**React styles**

A minor thing, but I noticed the filename `ratings.jsx` is lower case and the rest are Pascal case. I've been using all lower case but realized that's probably weird, so I googled and found that React has no opinion on it, but it's more common to Pascal, and AirBnB does in their style guide:

https://airbnb.io/javascript/react/

If everyone else is cool with it, we could conform to ^this guide to keep everything looking like one team wrote it. If so, all our JSX filenames will need to be Pascal case.

aboussarath commented:
> Is there a reason for the outer `<div>`? If not, you might want to remove it, make the ratings-reviews `<div>` the top level, and move the `<p>` inside it.

I wrote my divs like that as a quick way to keep my title separate from my content layout. I just made a div to wrap it all together because initially my styling was putting everything on the same row. 
<img width="805" alt="Screen Shot 2021-06-06 at 12 41 43 PM" src="https://user-images.githubusercontent.com/60486755/121025847-36380080-c76b-11eb-8d84-e781aa1ed421.png">
I agree that's how our layout should be in app and I refactored my code. Hopefully it looks a little bit cleaner!

> A minor thing, but I noticed the filename `ratings.jsx` is lower case and the rest are Pascal case. I've been using all lower case but realized that's probably weird, so I googled and found that React has no opinion on it, but it's more common to Pascal, and AirBnB does in their style guide:

I'm cool with following the AirBnB style guide. I went ahead and changed `ratings.jsx` to `Reviews.jsx` to be more consistent with how this widget is named in our deliverables tracker. I was keeping that file lowercase in order to avoid merge conflicts, but it did look a little weird the way it was. There is also an ESLint config for the AirBnB styling if we want to go that route - https://www.npmjs.com/package/eslint-config-airbnb
